/home/nianliu/ModelGarden/models-r2.1.0/official/modeling/hyperparams/params_dict.py:402: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  params_dict = yaml.load(dict_or_string_or_yaml_file)
I0817 01:40:45.612145 139743174657856 main.py:195] Model Parameters: {'anchor': {'anchor_size': 4.0,
            'aspect_ratios': [1.0, 2.0, 0.5],
            'max_level': 7,
            'min_level': 3,
            'num_scales': 3},
 'architecture': {'backbone': 'resnet',
                  'multilevel_features': 'fpn',
                  'parser': 'retinanet_parser',
                  'use_bfloat16': False},
 'enable_summary': False,
 'eval': {'batch_size': 8,
          'eval_file_pattern': 'gs://apiss/coco/val-*',
          'eval_samples': 1000,
          'eval_timeout': None,
          'input_sharding': True,
          'min_eval_interval': 180,
          'type': 'box',
          'val_json_file': 'gs://apiss/coco/instances_val2017.json'},
 'fpn': {'batch_norm': {'batch_norm_epsilon': 0.0001,
                        'batch_norm_momentum': 0.997,
                        'batch_norm_trainable': True},
         'fpn_feat_dims': 256,
         'max_level': 7,
         'min_level': 3,
         'use_separable_conv': False},
 'model_dir': 'gs://apiss/coco/Model/cocopy_func9',
 'nasfpn': {'batch_norm': {'batch_norm_epsilon': 0.0001,
                           'batch_norm_momentum': 0.997,
                           'batch_norm_trainable': True},
            'dropblock': {'dropblock_keep_prob': None, 'dropblock_size': None},
            'fpn_feat_dims': 256,
            'max_level': 7,
            'min_level': 3,
            'num_repeats': 5,
            'use_separable_conv': False},
 'postprocess': {'max_level': 7,
                 'max_total_size': 100,
                 'min_level': 3,
                 'nms_iou_threshold': 0.5,
                 'pre_nms_num_boxes': 5000,
                 'score_threshold': 0.05,
                 'use_batched_nms': False},
 'predict': {'predict_batch_size': 8},
 'resnet': {'batch_norm': {'batch_norm_epsilon': 0.0001,
                           'batch_norm_momentum': 0.997,
                           'batch_norm_trainable': True},
            'dropblock': {'dropblock_keep_prob': None, 'dropblock_size': None},
            'resnet_depth': 50},
 'retinanet_head': {'anchors_per_location': 9,
                    'batch_norm': {'batch_norm_epsilon': 0.0001,
                                   'batch_norm_momentum': 0.997,
                                   'batch_norm_trainable': True},
                    'max_level': 7,
                    'min_level': 3,
                    'num_classes': 91,
                    'retinanet_head_num_convs': 4,
                    'retinanet_head_num_filters': 256,
                    'use_separable_conv': False},
 'retinanet_loss': {'box_loss_weight': 50,
                    'focal_loss_alpha': 0.25,
                    'focal_loss_gamma': 1.5,
                    'huber_loss_delta': 0.1,
                    'num_classes': 91},
 'retinanet_parser': {'aug_rand_hflip': True,
                      'aug_scale_max': 1.0,
                      'aug_scale_min': 1.0,
                      'autoaugment_policy_name': 'v0',
                      'match_threshold': 0.5,
                      'max_num_instances': 100,
                      'num_channels': 3,
                      'output_size': [640, 640],
                      'skip_crowd_during_training': True,
                      'unmatched_threshold': 0.5,
                      'use_autoaugment': False,
                      'use_bfloat16': False},
 'strategy_config': {'all_reduce_alg': None,
                     'num_gpus': 0,
                     'num_packs': 1,
                     'task_index': -1,
                     'tpu': 'node-1',
                     'worker_hosts': None},
 'strategy_type': 'tpu',
 'train': {'batch_size': 64,
           'checkpoint': {'path': '', 'prefix': ''},
           'frozen_variable_prefix': '(resnet\\d+/)conv2d(|_([1-9]|10))\\/',
           'input_sharding': False,
           'iterations_per_loop': 500,
           'l2_weight_decay': 0.0001,
           'learning_rate': {'init_learning_rate': 0.08,
                             'learning_rate_levels': [0.008, 0.0008],
                             'learning_rate_steps': [15000, 20000],
                             'type': 'step',
                             'warmup_learning_rate': 0.0067,
                             'warmup_steps': 500},
           'optimizer': {'momentum': 0.9, 'nesterov': True, 'type': 'momentum'},
           'total_steps': 22500,
           'train_file_pattern': '',
           'transpose_input': False},
 'type': 'retinanet',
 'use_tpu': True}
I0817 01:40:45.657138 139743174657856 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:40:45.699565 139743174657856 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:40:45.699865 139743174657856 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:40:45.700048 139743174657856 client.py:777] Refreshing access_token
I0817 01:40:45.788547 139743174657856 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:40:45.840035 139743174657856 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:40:45.840216 139743174657856 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:40:45.840323 139743174657856 client.py:777] Refreshing access_token
I0817 01:40:46.020675 139743174657856 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:40:46.056772 139743174657856 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:40:46.057077 139743174657856 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:40:46.057231 139743174657856 client.py:777] Refreshing access_token
I0817 01:40:46.155856 139743174657856 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:40:46.205586 139743174657856 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:40:46.205760 139743174657856 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:40:46.205863 139743174657856 client.py:777] Refreshing access_token
I0817 01:40:46.297041 139743174657856 remote.py:177] Entering into master device scope: /job:worker/replica:0/task:0/device:CPU:0
2021-08-17 01:40:46.297490: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2021-08-17 01:40:46.302581: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2249995000 Hz
2021-08-17 01:40:46.302884: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x501e3a0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-08-17 01:40:46.302911: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-08-17 01:40:46.307315: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job worker -> {0 -> 10.116.237.58:8470}
2021-08-17 01:40:46.307410: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job localhost -> {0 -> localhost:42559}
2021-08-17 01:40:46.323074: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job worker -> {0 -> 10.116.237.58:8470}
2021-08-17 01:40:46.323132: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job localhost -> {0 -> localhost:42559}
2021-08-17 01:40:46.323607: I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:390] Started server with target: grpc://localhost:42559
INFO:tensorflow:Initializing the TPU system: node-1
I0817 01:40:46.324352 139743174657856 tpu_strategy_util.py:72] Initializing the TPU system: node-1
INFO:tensorflow:Clearing out eager caches
I0817 01:40:46.489620 139743174657856 tpu_strategy_util.py:100] Clearing out eager caches
INFO:tensorflow:Finished initializing TPU system.
I0817 01:40:51.361971 139743174657856 tpu_strategy_util.py:123] Finished initializing TPU system.
I0817 01:40:51.363544 139743174657856 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:40:51.400466 139743174657856 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:40:51.400815 139743174657856 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:40:51.400972 139743174657856 client.py:777] Refreshing access_token
I0817 01:40:51.487587 139743174657856 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:40:51.525403 139743174657856 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:40:51.525640 139743174657856 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:40:51.525776 139743174657856 client.py:777] Refreshing access_token
INFO:tensorflow:Found TPU system:
I0817 01:40:51.626058 139743174657856 tpu_system_metadata.py:140] Found TPU system:
INFO:tensorflow:*** Num TPU Cores: 8
I0817 01:40:51.626392 139743174657856 tpu_system_metadata.py:141] *** Num TPU Cores: 8
INFO:tensorflow:*** Num TPU Workers: 1
I0817 01:40:51.626538 139743174657856 tpu_system_metadata.py:142] *** Num TPU Workers: 1
INFO:tensorflow:*** Num TPU Cores Per Worker: 8
I0817 01:40:51.626670 139743174657856 tpu_system_metadata.py:144] *** Num TPU Cores Per Worker: 8
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:CPU:0, CPU, 0, 0)
I0817 01:40:51.626841 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:CPU:0, CPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
I0817 01:40:51.627098 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:CPU:0, CPU, 0, 0)
I0817 01:40:51.627244 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:CPU:0, CPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:0, TPU, 0, 0)
I0817 01:40:51.627471 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:0, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:1, TPU, 0, 0)
I0817 01:40:51.627616 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:1, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:2, TPU, 0, 0)
I0817 01:40:51.627772 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:2, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:3, TPU, 0, 0)
I0817 01:40:51.627913 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:3, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:4, TPU, 0, 0)
I0817 01:40:51.628071 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:4, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:5, TPU, 0, 0)
I0817 01:40:51.628211 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:5, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:6, TPU, 0, 0)
I0817 01:40:51.628351 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:6, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:7, TPU, 0, 0)
I0817 01:40:51.628491 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:7, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU_SYSTEM:0, TPU_SYSTEM, 0, 0)
I0817 01:40:51.628626 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU_SYSTEM:0, TPU_SYSTEM, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
I0817 01:40:51.628772 139743174657856 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
I0817 01:40:51.629344 139743174657856 main.py:135] Eval num_replicas_in_sync 8 num_workers 1 is_multi_host False
I0817 01:41:21.324950 139743174657856 distributed_executor.py:607] Starting to evaluate.
I0817 01:41:22.337890 139743174657856 distributed_executor.py:615] Checkpoint file gs://apiss/coco/Model/cocopy_func9/ctl_step_5000.ckpt-10 found and restoring from checkpoint
WARNING:tensorflow:AutoGraph could not transform <function InputFn.__call__.<locals>.<lambda> at 0x7f17ded00f28> and will run it as-is.
Cause: Unable to identify source code of lambda function <function InputFn.__call__.<locals>.<lambda> at 0x7f17ded00f28>. It was defined on this line: from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
map_func, which must contain a single lambda with matching signature. To avoid ambiguity, define each lambda in a separate expression.
W0817 01:41:27.930584 139743174657856 ag_logging.py:146] AutoGraph could not transform <function InputFn.__call__.<locals>.<lambda> at 0x7f17ded00f28> and will run it as-is.
Cause: Unable to identify source code of lambda function <function InputFn.__call__.<locals>.<lambda> at 0x7f17ded00f28>. It was defined on this line: from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
map_func, which must contain a single lambda with matching signature. To avoid ambiguity, define each lambda in a separate expression.
I0817 01:41:34.739090 139743174657856 detection_executor.py:123] Running evaluation after step: 5000.
I0817 01:44:21.000487 139743174657856 coco_evaluator.py:151] Using annotation file: gs://apiss/coco/instances_val2017.json
I0817 01:44:45.313086 139743174657856 detection_executor.py:137] Step: [5000] Validation metric = {'AP': 0.23233627, 'AP50': 0.38573697, 'AP75': 0.24353963, 'APs': 0.10166318, 'APm': 0.30357513, 'APl': 0.3539066, 'ARmax1': 0.22077253, 'ARmax10': 0.35697532, 'ARmax100': 0.3767283, 'ARs': 0.14376085, 'ARm': 0.45308077, 'ARl': 0.542652}
2021-08-17 01:44:45.313517: W tensorflow/core/distributed_runtime/eager/remote_tensor_handle_data.cc:75] Unable to destroy remote tensor handles. If you are running a tf.function, it usually indicates some op in the graph gets an error: 4 root error(s) found.
  (0) Cancelled: {{function_node __inference_test_step_89460}} Function was cancelled before it was started
  (1) Cancelled: {{function_node __inference_test_step_89460}} Function was cancelled before it was started
  (2) Out of range: {{function_node __inference_test_step_89460}} End of sequence
	 [[{{node IteratorGetNext_5}}]]
  (3) Out of range: {{function_node __inference_test_step_89460}} End of sequence
	 [[{{node IteratorGetNext_6}}]]
0 successful operations.
5 derived errors ignored.
I0817 01:44:45.315822 139743174657856 distributed_executor.py:622] Step: 5000 evalation metric = {'AP': 0.23233627, 'AP50': 0.38573697, 'AP75': 0.24353963, 'APs': 0.10166318, 'APm': 0.30357513, 'APl': 0.3539066, 'ARmax1': 0.22077253, 'ARmax10': 0.35697532, 'ARmax100': 0.3767283, 'ARs': 0.14376085, 'ARm': 0.45308077, 'ARl': 0.542652}.
I0817 01:44:46.239917 139743174657856 main.py:168] Final eval metric AP: 0.232336
I0817 01:44:46.240165 139743174657856 main.py:168] Final eval metric AP50: 0.385737
I0817 01:44:46.240266 139743174657856 main.py:168] Final eval metric AP75: 0.243540
I0817 01:44:46.240361 139743174657856 main.py:168] Final eval metric APs: 0.101663
I0817 01:44:46.240458 139743174657856 main.py:168] Final eval metric APm: 0.303575
I0817 01:44:46.240534 139743174657856 main.py:168] Final eval metric APl: 0.353907
I0817 01:44:46.240626 139743174657856 main.py:168] Final eval metric ARmax1: 0.220773
I0817 01:44:46.240698 139743174657856 main.py:168] Final eval metric ARmax10: 0.356975
I0817 01:44:46.240799 139743174657856 main.py:168] Final eval metric ARmax100: 0.376728
I0817 01:44:46.241146 139743174657856 main.py:168] Final eval metric ARs: 0.143761
I0817 01:44:46.241233 139743174657856 main.py:168] Final eval metric ARm: 0.453081
I0817 01:44:46.241323 139743174657856 main.py:168] Final eval metric ARl: 0.542652
loading annotations into memory...
Done (t=0.48s)
creating index...
index created!
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=17.91s).
Accumulating evaluation results...
DONE (t=1.92s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.232
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.386
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.244
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.102
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.304
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.354
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.221
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.357
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.377
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.144
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.453
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.543
2021-08-17 01:44:47.403615: W ./tensorflow/core/distributed_runtime/eager/destroy_tensor_handle_node.h:79] Ignoring an error encountered when deleting remote tensors handles: Invalid argument: Unable to find a context_id matching the specified one (13672806834919838737). Perhaps the worker was restarted, or the context was GC'd?
Additional GRPC error information:
{"created":"@1629164687.403537453","description":"Error received from peer","file":"external/grpc/src/core/lib/surface/call.cc","file_line":1039,"grpc_message":"Unable to find a context_id matching the specified one (13672806834919838737). Perhaps the worker was restarted, or the context was GC'd?","grpc_status":3}
